bluecellulab-bb5-tests/README.md

```diff
@@ -0,0 +1,60 @@
+# bluecellulab-bb5-tests
+
+## Description
+
+This folder contains tests for bluecellulab on the large circuits located on bb5. Please note that these tests are not directly usable as they are due to dependencies.
```

Comment:
Could you add a comma here?
not directly usable as they are, due to dependencies